cancer_ai/validator/models.py

```diff
@@ -52,4 +52,16 @@ def find_organization_by_competition_id(self, competition_id: str) -> Optional[O
 class NewDatasetFile(BaseModel):
     competition_id: str = Field(..., min_length=1, description="Competition identifier")
     dataset_hf_repo: str = Field(..., min_length=1, description="Hugging Face repository path for the dataset")
-    dataset_hf_filename: str = Field(..., min_length=1, description="Filename for the dataset in the repository")
\ No newline at end of file
+    dataset_hf_filename: str = Field(..., min_length=1, description="Filename for the dataset in the repository")
+
+
+
+# TODO
```

Comment:
I think we were considering abstracting wandb api further but we agreed at some point that the api is api enough itself?

Reply:
I did it already